packages/angular/src/generators/init/lib/ensure-peer-deps.ts

```diff
@@ -0,0 +1,44 @@
+import { ensurePackage, Tree } from '@nrwl/devkit';
+import * as moduleAlias from 'module-alias';
+import { angularVersion } from '../../../utils/versions';
+
+export async function ensurePeersDeps(tree: Tree) {
```

Comment:
I'm not sure we should do this. When creating new workspaces with Angular, we install those peerDependencies as part of the Angular preset (`new` generator). When migrating from Angular CLI workspaces, we also ensure those peerDependencies are added. Those are scenarios we are "in charge", and we are expected to handle every requirement to either create a new workspace or migrate an Angular CLI one to Nx.

The only scenario we are not automatically handling (and the only scenario this utility would address) is when someone manually installs the `@nrwl/angular` plugin to an existing workspace. In that case, the package manager will report the missing peerDependencies and the user should install them. I don't think we should automate the package manager work in that case and we could avoid the hacks below by not doing so.

Reply:
removed this filed